### PR TITLE
Use USERPROFILE dir to store ccache on Windows.

### PR DESCRIPTION
**Pull request by nmoinvaz**
See #80.

### CODE REVIEW

File: src/restore.ts [hunk 1]
```diff
@@ -70,8 +70,7 @@ async function installCcacheWindows() : Promise<void> {
     "4.6.1",
     "windows-x86_64",
     "bcf4b82faa648ff77dacc11124be2858d48064bae4096d26b396e2846cc22da3",
-    // TODO find a better place
-    "C:\\Users\\runneradmin\\.cargo\\bin",
+    `${process.env.USERPROFILE}\\.cargo\\bin`,
```
Comment (jonashaag) [inline src/restore.ts:73]:
```suggestion
    // TODO find a better place
    `${process.env.USERPROFILE}\\.cargo\\bin`,
```

Still a bad place.

File: src/restore.ts [hunk 2]
```diff
@@ -95,8 +94,7 @@ async function installSccacheWindows() : Promise<void> {
     "v0.3.0",
     "x86_64-pc-windows-msvc",
     "e58646b1c377c7f873c7e13ae7298631fb490703ee15ad67ce326ed0dde2067e",
-    // TODO find a better place
-    "C:\\Users\\runneradmin\\.cargo\\bin",
+    `${process.env.USERPROFILE}\\.cargo\\bin`,
```
Comment (jonashaag) [inline src/restore.ts:97]:
```suggestion
    // TODO find a better place
    `${process.env.USERPROFILE}\\.cargo\\bin`,
```

**Review by jonashaag — CHANGES_REQUESTED** [review]

**Review by jonashaag — APPROVED** [review]

**Comment by nmoinvaz** [comment]
It looks like PATH environment variable needs to be set.

**Comment by jonashaag** [comment]
Is this on a self-hosted runner or on a GitHub runner? On GitHub I would expect `.cargo` to be in `PATH`. We can also find another place to put the file, or use another installation method like `choco` (I don't know anything about `choco` though)

**Comment by nmoinvaz** [comment]
Yes, it is on a self-hosted runner. `choco` is a popular package manager for Windows that is also pre-installed on GitHub's runners. You just run `choco install ccache` and it will install `ccache` and put it in the `PATH` - similar to `brew install ccache` etc.

**Comment by nmoinvaz** [comment]
No worries. I didn't push hard for it because I still think `choco` is probably the way to go on Windows.

**Pull request merged**